### PR TITLE
Fix PP renaming subtitle to .srt.srt

### PR DESCRIPTION
**Pull request by fernandog**
Fixes https://github.com/pymedusa/Medusa/issues/3489
@medariox when calling from_ietf_code with `srt`, it returns an alpha3 code with `srt` in Language object but no country information 

IMO if no country, so the Language object is not a valid language, so we should not use it. 
Let me know what do you think

This issue was introduced recently as moving a SRT with no language wasn't a issue since I first started to used subtitles in Medusa/SR

### CODE REVIEW

**Comment by medariox** [comment]
Isn't this more a babelfish issue then? `srt` isn't a valid alpha3 code, right? Why is it returning a result in the first place?

**Comment by fernandog** [comment]
Well, it's returning a Language object to be consistent but they set None as country

**Comment by medariox** [comment]
How is that consistent? From the code and documentation, it should raise `ValueError('%r is not a valid language' % language)`: https://github.com/Diaoul/babelfish/blob/master/babelfish/language.py#L75-L82

**Comment by fernandog** [comment]
srt is a valid Language btw
https://github.com/Diaoul/babelfish/blob/master/babelfish/data/iso-639-3.tab#L6047

called here: https://github.com/Diaoul/babelfish/blob/master/babelfish/language.py#L26-L30

if you have a better fix close this then

File: medusa/post_processor.py [hunk 1]
```diff
@@ -367,14 +367,16 @@ def rename_associated_file(new_path, new_basename, filepath):
...
-                    code = 'pt-BR'
-                new_extension = code + '.' + extension
-                extension = sub_code + '.' + extension
+            sub_code = filepath.rsplit('.', 2)
+            # len != 3 means we have a subtitle without language
+            if len(sub_code) == 3:
+                code = sub_code[1].lower().replace('_', '-')
+                if from_code(code, unknown='') or from_ietf_code(code, unknown=''):
+                    # TODO remove this hardcoded language
+                    if code == 'pt-br':
```
Comment (fernandog) [inline medusa/post_processor.py]: pt-br > 3 ?

Comment (medariox) [reply medusa/post_processor.py]: Nope, that's something different. Try to `filepath.rsplit('.', 2)` and you will understand what's going on.

Comment (fernandog) [reply medusa/post_processor.py]: we also support `en` , `pt`, and `it` and language codes

Comment (medariox) [reply medusa/post_processor.py]: That should be fine, see the tests.

**Pull request merged**